### PR TITLE
Fix checking for script setup in vue/require-typed-ref

### PR DESCRIPTION
**Pull request by kovalchukq**

### CODE REVIEW

File: tests/lib/rules/require-typed-ref.js
Comment (waynzh) [inline tests/lib/rules/require-typed-ref.js:1]: Perhaps it also needs a invalid test case for this `export default setup`? Something like:
```
<script lang="ts">
  import { ref } from 'vue'
  export default {
    setup() {
      const count = ref()
    }
  }
</script>
```

Comment (kovalchukq) [reply tests/lib/rules/require-typed-ref.js:1]: Fixed

**Review by FloEdelmann — COMMENTED** [review]
Thanks! Could you please fix the lint error (run `npm run lint:fix`)? Otherwise, this looks good to me.

**Comment by ota-meshi** [comment]
Hmm... What is this PR fixing? Could you please explain the details?

**Comment by kovalchukq** [comment]
> Hmm... What is this PR fixing? Could you please explain the details?
This PR skip checking `require-typed-ref ` rule for vue files that doesn't have lang="ts".

**Review by ota-meshi — APPROVED** [review]
Sorry for the slow response.
Thank you for your explanation. 
LGTM! Thank you!

**Review by SamyDgz — APPROVED** [review]

**Pull request merged**